things/things.go

```diff
@@ -33,6 +33,9 @@ type Service interface {
 	// the provided key.
 	ListClientsByGroup(ctx context.Context, token, groupID string, pm clients.Page) (clients.MembersPage, error)
 
+	// VerifyConnections verifies if a list of channels is connected to a list of channels.
+	VerifyConnections(ctx context.Context, token string, thingID, groupID []string) (clients.ConnectionsPage, error)
```

Comment:
instead of `thingID` `groupID` we can have like `thingIDs` and `groupIDs`, since they are slice